<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendSet</class>
 <widget class="QWidget" name="SendSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>20</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Lenth</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="sb_lenth">
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Method</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="r_static">
        <property name="text">
         <string>static</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="r_step">
        <property name="text">
         <string>step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="r_sum">
        <property name="text">
         <string>sum</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="le_name">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="3">
    <widget class="QSpinBox" name="sb_interval">
     <property name="suffix">
      <string>ms</string>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Interval</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Byte</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>200</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="7">
    <widget class="QPushButton" name="pb_del">
     <property name="text">
      <string>Del</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="pb_save">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3" colspan="5">
    <widget class="QStackedWidget" name="st_set">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="static_page">
      <widget class="QGroupBox" name="gridGroupBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>251</width>
         <height>171</height>
        </rect>
       </property>
       <property name="title">
        <string>Static setting</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>static</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="le_static"/>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="step_page">
      <widget class="QGroupBox" name="gridGroupBox_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>251</width>
         <height>161</height>
        </rect>
       </property>
       <property name="title">
        <string>Step setting</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>end</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>start</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="r_dec">
          <property name="text">
           <string>Dec</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="le_step_end"/>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>step</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="le_step_start"/>
        </item>
        <item row="2" column="3">
         <widget class="QLineEdit" name="le_step_step"/>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="r_inc">
          <property name="text">
           <string>Inc</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="4">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="3">
         <widget class="QComboBox" name="cb_over">
          <item>
           <property name="text">
            <string>STOP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LOOP</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>25</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>over</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="sum_page">
      <widget class="QGroupBox" name="gridGroupBox_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>251</width>
         <height>181</height>
        </rect>
       </property>
       <property name="title">
        <string>Sum setting</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="4" column="0">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>25</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="3">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>start</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>end</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="le_sum_end"/>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="le_sum_start"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="pb_add">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
